service/controller/release/resource/status/create.go

```diff
@@ -41,6 +43,40 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		}
 	}
 
+	// Doing this per-release isn't ideal, can we pass this list to each status reconcile somehow?
+	r.logger.LogCtx(ctx, "level", "debug", "message", "searching for running tenant clusters")
```

Comment:
Let's put this log message inside the block - similar to how it is below.

Reply:
Good idea